### PR TITLE
blog: add jake joins post

### PR DESCRIPTION
**Pull request by armon**

### CODE REVIEW

File: source/blog/2015-10-28-jake-joins-hashicorp.html.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+into our tools and solutions to better serve the whole community.
+
+READMORE
+
+At HashiCorp, we expect Jake to contribute across many parts of Atlas.
+Atlas is built using a service oriented designed, and is composed
+of dozens of microservices. As part of a focus on operational reliability,
+Jake will harden these services and improve our operational processes.
+
+Prior to HashiCorp, Jake worked at Minted where he worked on their
```
Comment (sethvargo) [inline source/blog/2015-10-28-jake-joins-hashicorp.html.md]: There are a lot of "priors" and "worked" in these two sentences

**Comment by sethvargo** [comment]
Imageoptim

File: source/blog/2015-10-28-jake-joins-hashicorp.html.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+page_title: "Jake Champlin Joins HashiCorp"
+title: "Jake Champlin Joins HashiCorp"
+list_image_url: /images/blog/jake-joins-hashicorp/list.jpg
+post_image_url: /images/blog/jake-joins-hashicorp/post.jpg
+author: Mitchell Hashimoto
```
Comment (sethvargo) [inline source/blog/2015-10-28-jake-joins-hashicorp.html.md]: Armon

File: source/blog/2015-10-28-jake-joins-hashicorp.html.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Jake will harden these services and improve our operational processes.
+
+Prior to HashiCorp, Jake worked at Minted where he worked on their
+application deployment process for their services. Prior to that,
+he worked at The Data Cave where he worked on automating their entire
+infrastructure. He is also a contributor to the kitchen-puppet driver
+for test kitchen.
+
+Jake is located in Columbus, Indiana where he lives with his wife
+and two dogs. Jake loves automation, learning, eating waffles, and
```
Comment (grubernaut) [inline source/blog/2015-10-28-jake-joins-hashicorp.html.md]: an aspiring

**Comment by grubernaut** [comment]
🍣🚀

**Pull request merged**